install.sh

```diff
@@ -103,6 +103,31 @@ EOF
     chmod +x "$root/$prefix/bin/$bin"
 }
 
+relocate_python3() {
+    local script="$1"
+    local scriptdir="$(dirname $script)"
```

Comment:
for extra safety, "$(dirname "$script")"

Reply:
fixed, force pushed.